### PR TITLE
New config to hide motion meta information box on projector (Fixed #3588)

### LINKED ISSUE #3588: New config to hide motion metadata box on projector

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -50,7 +50,8 @@ Motions:
 - Added multiselect filter for motion comments [#3372].
 - Added support for pinning personal notes to the window [#3360].
 - Clear identifier on state reset [#3356].
-- New config options to hide reason and recommendation on projector [#3432].
+- New config options to hide reason, recommendation and meta
+  information box on projector [#3432].
```
Comment (FinnStutzenstein) [inline CHANGELOG]: Please add this PR number to the entry.

File: openslides/motions/static/templates/motions/slide_motion.html [hunk 2]
```diff
@@ -1,5 +1,11 @@
+<style type="text/css" ng-if="!config('motions_disable_sidebox_on_projector')">
```
Comment (FinnStutzenstein) [inline openslides/motions/static/templates/motions/slide_motion.html]: In fact that the id `motioncontent` is only used once, please use a `ng-style`.

Comment (emanuelschuetze) [reply openslides/motions/static/templates/motions/slide_motion.html]: Done. Please review again.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review later.

**Pull request merged**